variables.tf

```diff
@@ -1137,12 +1137,7 @@ variable "s3_versioning_enabled" {
 variable "s3_lifecycle_rules" {
   description = "S3 Lifecycle rules"
   default     = []
-  type = list(object({
```

Comment:
This is wrong. The type should be explicit.

Reply:
why? i don´t see a reason for explicit type

Reply:
from my point of view it is better to have to know what data-structure is expected